packages/yarnpkg-libzip/sources/ZipFS.ts

```diff
@@ -135,6 +132,8 @@ export class ZipFS extends BasePortableFakeFS {
       }
     }
 
+    this.libzip = getInstance();
```

Comment:
Why have this as a member of the class?  The instance is constant once initialized, and this has the private modifier, so code that wants to extend ZipFS can’t access it anyway.  I feel like this could cause a memory leak in certain circumstances.

Reply:
Because otherwise I'd have to call `getInstance` in each function that uses the libzip, which isn't ideal